secretariat/views.py

```diff
@@ -7,3 +8,8 @@ def view_index(request):
 
 def view_accessibilite(request):
     return render(request, "secretariat/accessibilite.html")
+
+
+def logout_view(request):
```

Comment:
Suggestion : les deux autres vues commencent par `view_`, que penses-tu d'essayer de suivre le modèle ?

```suggestion
def view_logout(request):
```

Reply:
ah bah oui. Ce serait logique !